src/components/shared/copyable.js

```diff
@@ -11,6 +11,13 @@ class Copyable extends React.Component {
     copied: false,
   };
 
+  constructor(props) {
+    super(props);
+
+    this.copyToClipboard = this.copyToClipboard.bind(this);
+    this.mouseLeave = this.mouseLeave.bind(this);
```

Comment:
How about arrow notation for `copyToClipboard()`?

Reply:
I did a bit more research (also spurred by @wind0r thx!), and this appears to be the most performant way. 

It is more verbose, but it avoids some issues that the arrow notation brings. 

It's a micro optimisation, but I think we could apply this everywhere and then not have to think about it anymore. 

This is probably a `fmt` repo discussion, and I realise I just introduced a different way of doing something again. So I'd be fine with going for the arrow notation now, since we agreed a while back to use that in cases where a bind was actually needed, and then we can discuss this a bit more internally, and then possibly change it everywhere in one fell swoop.

Reply:
Yeah, talking about it makes sense. Otherwise we'll end up reverting each other's standardization efforts.

Reply:
I only noticed in the initial version of this pr that in there was a .bind() call in the render function which should be avoided :)

I don't think/know if there is any performance difference between Arrow Function or Bind in Constructor (the result should be quite the same, but i never checked the generated code). The important part just should be to avoid bind/arrow in render functions and that the solution should be the standardized/same around the code base

There also is an eslint rule for this: [react/jsx-no-bind](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-bind.md)